rubygem-hammer_cli_gutterball.spec

```diff
@@ -50,7 +49,8 @@ gem install --local --install-dir .%{gem_dir} \
 
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d
-install -m 755 %{SOURCE1} %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
+install -m 755 .%{geminstdir}/config/gutterball.yml %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
```

Comment:
we can change them all, its not too bad, but you also have to change all instances of 'gemname' to 'gem_name'  otherwise the 'gem_instdir' macro does not work.